raiden/api/rest.py

```diff
@@ -548,10 +567,10 @@ def register_token(
 
     def mint_token(
         self,
-        token_address: typing.TokenAddress,
-        to: typing.Address,
-        value: typing.TokenAmount,
-        contract_method: str,
+        token_address: TokenAddress,
+        to: Address,
+        value: TokenAmount,
+        contract_method: MintingMethod,
     ):
```

Comment:
missing return type

Reply:
Yes, I don't know the return type of the `rest` API and preferred to leave it without an annotation. I just wanted to add the `RaidenService` with this PR.

Reply:
Fine for me.